yang/frr-nexthop.yang

```diff
@@ -200,10 +204,45 @@ module frr-nexthop {
           "Identifies a nexthop within a nexthop group; the entries
            are ordered by id value, and the value has no other meaning.";
       }
+      uses frr-nexthop-attributes;
+    }
+  }
 
-      uses frr-nexthop;
+  typedef frr-nexthop-ref {
+    type leafref {
+      path "/frr-nexthop:frr-nexthop/frr-nexthop:entry/frr-nexthop:id";
     }
+    description
+      "Reference to a nexthop-group";
   }
 
-  // End of frr-nexthop-group
+  container frr-nexthop-group {
+    description
+      "A nexthop-group, represented as a list of nexthop objects.";
+
+    list group-entry {
+      key "group-id";
+      description
+        "A list of nexthop objects.";
+      leaf group-id {
+        type uint32;
+        description
+          "Identifies a nexthop within a nexthop group; the entries
+           are ordered by id value, and the value has no other meaning.";
```

Comment:
The description text here doesn't seem right - it seems to be talking about a single nexthop, rather than a group. Also, the id value in a 'real' group does have some meaning - at least, it can be used when interacting with the newer linux kernels.

Reply:
I will update the same.